test/parallel_api/algorithm/alg.modifying.operations/shift_left.pass.cpp

```diff
@@ -0,0 +1,96 @@
+// -*- C++ -*-
+//===-- shift_left.pass.cpp -----------------------------------------------===//
+//
+// Copyright (C) 2021 Intel Corporation
```

Comment:
Please remove year from copyright (see #94) here and in other files.
@ValentinaKats, FYI